product_contract/tests/test_sale_order.py

```diff
@@ -20,6 +20,11 @@ def setUp(self):
         self.product.product_tmpl_id.is_contract = True
         self.product.product_tmpl_id.contract_template_id = self.contract.id
 
+    def tearDown(self):
+        self.env['account.analytic.account']._revert_method(
```

Comment:
Oh forget it, it's reverting https://github.com/OCA/contract/pull/72/files#diff-f8a2fa2a1e36a08f8fecab38bfbd417bR32; thanks!